src/buffer/buffer.rs

```diff
@@ -23,22 +26,22 @@ use crate::{buffer::Cell, prelude::*};
 ///     width: 10,
 ///     height: 5,
 /// });
-/// buf.get_mut(0, 2).set_symbol("x");
-/// assert_eq!(buf.get(0, 2).symbol(), "x");
+/// buf[(0, 2)].set_symbol("x");
+/// assert_eq!(buf[(0, 2)].symbol(), "x");
```

Comment:
Maybe add an example of cell(_mut) here to highlight both ways of accessing cells? Probably with `?` in that case.

Reply:
d2321a458